ios/FluentUI/Vnext/List/List.swift

```diff
@@ -35,15 +36,13 @@ public struct MSFListView: View {
             ScrollView {
                 VStack(spacing: 0) {
                     ForEach(sections, id: \.self) { section in
-                        if let sectionTitle = section.title {
-                            Header(title: sectionTitle, tokens: tokens)
+                        if let sectionTitle = section.title, !sectionTitle.isEmpty {
+                            Header(state: section, windowProvider: tokens.windowProvider)
                         }
 
                         ForEach(section.cells.indices, id: \.self) { index in
                             let cellState = section.cells[index]
-                            let hasDividers = (index < section.cells.count - 1 && section.hasDividers) || (section.hasDividers && (cellState.children != nil))
```

Comment:
Related to my other comment, setting the cellState.hasDivider seems like a better option compared to leaving that up to the client.
We can sync offline about it.